input/resources/au-core-allergyintolerance.xml

```diff
@@ -78,6 +78,21 @@
       <path value="AllergyIntolerance.code"/>
       <min value="1"/>
       <mustSupport value="true"/>
+      <binding>
+        <extension url="http://hl7.org/fhir/tools/StructureDefinition/additional-binding">
+          <extension url="purpose">
+            <valueCode value="candidate"/>
+          </extension>
+          <extension url="valueSet">
+            <valueCanonical value="https://healthterminologies.gov.au/fhir/ValueSet/adverse-reaction-substances-and-negated-findings-1"/>
+          </extension>
+          <extension url="documentation">
```

Comment:
This documentation is fairly pointless in terms of additional binding documentation. I accept that we will keep for now and raise a change request to fix later since this is a description of the value set not the documentation of the additional binding.